ReSharper.FSharp/test/data/features/generate/overrides/Anchor - Repr 13.fs.gold

```diff
@@ -0,0 +1,16 @@
+﻿Provided elements:
+ 0: ToString():System.String
+ 1: Equals(System.Object):System.Boolean
+ 2: GetHashCode():System.Int32
+ 3: Finalize():System.Void
+
+// ${KIND:Overrides}
+// ${SELECT0:ToString():System.String}
+
+// ${KIND:Overrides}
+
+type C() = 
+  class
+    override this.ToString() = {selstart}failwith "todo"{selend}
+    let f = 23
```

Comment:
This order is incorrect.

Reply:
I see that it's going to be addressed separately (https://github.com/JetBrains/resharper-fsharp/pull/512#discussion_r1169020142).